docs/client.rst

```diff
@@ -79,7 +79,7 @@ that case you can specify multiple values for each key::
                            params=params) as r:
         assert r.url == 'http://httpbin.org/get?key=value2&key=value1'
 
-You can also pass :class:`str` content as param, but beware - content
+You can also pass :class:`str` content as param, but beware -- content
```

Comment:
"-" basically looks to short and double dashes converted into a real wide dash "–"